client/src/components/SectionTable/SectionTable.js

```diff
@@ -79,6 +80,28 @@ class SectionTable extends PureComponent {
                         <Fragment />
                     )}
                 </div>
+
+                <div
+                    style={{
+                        display: 'inline-flex',
+                        marginTop: '0px',
+                        fontSize: '15px',
+                    }}
+                >
+                    <a
+                        target="blank"
+                        href={
+                            'https://zotistics.com/?&selectQuarter=&selectYear=&selectDep=' +
+                            urlEncode +
+                            '&classNum=' +
+                            this.props.courseDetails.courseNumber +
+                            '&code=&submit=Submit'
+                        }
+                    >
+                        Zotistics
+                    </a>
```

Comment:
Let's move this `<a>` tag into the div above it, on line 81. Then we won't need the `div` that it's currently wrapped inside of.

Reply:
sounds good, I'll get into that